Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/django_patterns'...
Listing './lib/python3.5/site-packages/django_patterns/db'...
Listing './lib/python3.5/site-packages/django_patterns/db/fields'...
Listing './lib/python3.5/site-packages/django_patterns/db/fields/base64_encoded_test'...
Listing './lib/python3.5/site-packages/django_patterns/db/fields/uuid_field_test'...
Listing './lib/python3.5/site-packages/django_patterns/db/models'...
Listing './lib/python3.5/site-packages/django_patterns/db/models/mixins'...
Compiling './lib/python3.5/site-packages/django_patterns/db/models/mixins/positional_order.py'...
***   File "./lib/python3.5/site-packages/django_patterns/db/models/mixins/positional_order.py", line 89
    raise ValueError, _(u"‘self’ cannot be combined with other fields in order_with_respect_to expression.")
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/django_patterns/db/models/mixins/positional_order_test'...
Listing './lib/python3.5/site-packages/django_patterns/db/models/mixins/uuid_primary_key_test'...
Listing './lib/python3.5/site-packages/django_patterns/db/models/mixins/uuid_stamped_test'...
Listing './lib/python3.5/site-packages/django_patterns/management'...
Listing './lib/python3.5/site-packages/django_patterns/management/commands'...
Compiling './lib/python3.5/site-packages/django_patterns/management/commands/discover_unittest_modules.py'...
***   File "./lib/python3.5/site-packages/django_patterns/management/commands/discover_unittest_modules.py", line 104
    print "\n".join(apps)
             ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/django_patterns/templatetags'...
Compiling './lib/python3.5/site-packages/django_patterns/templatetags/macros.py'...
***   File "./lib/python3.5/site-packages/django_patterns/templatetags/macros.py", line 129
    raise template.TemplateSyntaxError, "'%s' tag requires at least one argument (macro name)" % token.contents.split()[0]
                                      ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/django_patterns/test'...
Listing './lib/python3.5/site-packages/django_patterns/test/project'...
Listing './lib/python3.5/site-packages/django_patterns-0.0.3.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
